### PR TITLE
Improvement: support embedded resources & changes_only tracking

### PR DESCRIPTION
**Pull request by rgraff**

### CODE REVIEW

**Comment by rgraff** [comment]
Fixes #9 by using Ash.Type.dump_to_embedded

File: README.md [hunk 1]
```diff
@@ -45,6 +48,12 @@ use Ash.Registry,
   ]
 ```
 
+Lastly, you need to the change_tracking_mode. This can be either :snapshot or :changes_only
+
+```
+config :ash_paper_trail, :change_tracking_mode, :changes_only
```
Comment (zachdaniel) [inline README.md:54]: I think we should probably make this a config in the DSL

**Review by zachdaniel — CHANGES_REQUESTED** [review]
Looks great! I think the config should be done in the resource. It adds some duplication, but global config also makes it rigid to configure.

**Comment by rgraff** [comment]
@zachdaniel let me know if that works and I can do a squash commit to clean the history

**Review by zachdaniel — APPROVED** [review]

**Comment by zachdaniel** [comment]
LGTM!

**Comment by rgraff** [comment]
Ready to 🚢

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**